cwclient/cwclient.go

```diff
@@ -0,0 +1,172 @@
+package cwclient
+
+import (
+	"context"
+	"encoding/json"
+	"time"
+
+	wasmtypes "github.com/CosmWasm/wasmd/x/wasm/types"
+	"github.com/babylonlabs-io/finality-gadget/types"
+	rpcclient "github.com/cometbft/cometbft/rpc/client"
+	cosmosclient "github.com/cosmos/cosmos-sdk/client"
+)
+
+type CosmWasmClient struct {
+	rpcclient.Client
+	contractAddr string
```

Comment:
do you think we need a config struct for cw client as well?

Reply:
since it's a simple query client and there is only `contractAddr` needed, a config struct might be unnecessary?